lib/gui/modules/analytics.js

```diff
@@ -39,6 +40,22 @@ const analytics = angular.module(MODULE_NAME, [
   require('../models/settings')
 ]);
 
+/**
+ * @summary Get OS architecture
+ *
+ * @description
+ * We need this because node's os.arch() returns the process architecture
+ * See: https://github.com/nodejs/node-v0.x-archive/issues/2862
+ *
+ * @return {String} OS architecture
+ */
+const getOSArch = () => {
```

Comment:
Hm, I didn't know detecting the host architecture would be so tricky. Based on my other comments on this same function, what about something like this?

```js
const getHostArchitecture = () => {
    return _.includes([ 'ia32', 'x64' ], process.arch) ? arch() : process.arch;
};
```

So if the node architecture is not x86 nor x64, we don't try to disambiguate and return that (e.g: in case of `arm`), otherwise, we attempt to find the right one using the `arch` module, which only works for those architectures.

I think its the best we can do :/